src/app/organisations/postEditOrganisation.js

```diff
@@ -0,0 +1,40 @@
+const { sendResult } = require("../../infrastructure/utils");
+const {
+  getOrganisationByIdV2,
+} = require("./../../infrastructure/organisations");
+
+const postEditOrganisation = async (req, res) => {
+  const organisation = await getOrganisationByIdV2(req.params.id, req.id);
+  const { name, address } = req.body;
+
+  const view = "organisations/views/editOrganisation";
```

Comment:
Really minor but this is a variable you only use for 1 thing.  Might as well put the string in the 1 place you use it below and save a line of needless code